x-element.js

```diff
@@ -1488,10 +1488,11 @@ class TemplateEngine {
         }
         const previousSibling = node.previousSibling;
         if (previousSibling === startNode) {
-          const textNode = document.createTextNode(value ?? '');
```

Comment:
This was the only place we _really_ needed to coalesce. If you pass `null` or `undefined` here you get `'null'` or `'undefined'`.